maker/maker.go

```diff
@@ -117,6 +148,15 @@ func MakeInterface(comment, pkgName, ifaceName, ifaceComment string, methods []s
 	return FormatCode(code)
 }
 
+// isFunctionPrivate checks whether the starting
+// sign is lower case
+func isFunctionPrivate(name string) bool {
```

Comment:
+1, it definitely makes it more readable. I also wonder if go compiler actually recognizes capital and lower cases of other languages. So far, this tool is English only.